internal/db/dbutil/dbutil.go

```diff
@@ -153,6 +160,15 @@ func MigrateDB(db *sql.DB, dataSource string) error {
 	return err
 }
 
+type stdoutLogger struct{}
+
+func (logger stdoutLogger) Printf(format string, v ...interface{}) {
```

Comment:
Suggestion: When a method doesn't use its receiver, the name of the receiver may be omitted
```suggestion
func (stdoutLogger) Printf(format string, v ...interface{}) {
```

This is a helpful signal for the reader too.

Even when it's not omitted, a [shorter receiver name](https://github.com/golang/go/wiki/CodeReviewComments#receiver-names) is probably preferable.

Reply:
That is a helpful signal, I like that.